packages/core/src/shared/handleUninstall.ts

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { join } from 'path'
+import { getLogger } from './logger/logger'
+import { telemetry } from './telemetry'
+import { VSCODE_EXTENSION_ID } from './extensions'
+import { extensionVersion } from './vscode/env'
+
```

Comment:
Can we add documentation which links to the original implementation and note how this is a hack.
If this implementation stops working, future devs will be able to look at the other project's work and hopefully they will have come up with another solution.